npm/ng-packs/packages/theme-shared/src/lib/theme-shared.module.ts

```diff
@@ -38,7 +39,11 @@ export function appendScript(injector: Injector) {
 }
 
 @NgModule({
-  imports: [CoreModule, NgxValidateCoreModule],
+  imports: [
+    CoreModule,
+    NgxValidateCoreModule,
+    NgxsModule.forRoot([])
```

Comment:
I think NgxsModule is not necessary.

Reply:
If I don't add this I get 

main.ts:13 NullInjectorError: R3InjectorError[ThemeSharedModule -> ErrorHandler -> Actions -> Actions -> Actions]: 
  NullInjectorError: No provider for Actions!
    at NullInjector.get (http://localhost:4200/vendor.js:23662:27)
    at R3Injector.get (http://localhost:4200/vendor.js:37385:33)
    at R3Injector.get (http://localhost:4200/vendor.js:37385:33)
    at R3Injector.get (http://localhost:4200/vendor.js:37385:33)
    at injectInjectorOnly (http://localhost:4200/vendor.js:23517:33)
    at Module.ɵɵinject (http://localhost:4200/vendor.js:23527:57)
    at Object.ErrorHandler_Factory [as factory] (http://localhost:4200/main.js:9952:141)
    at R3Injector.hydrate (http://localhost:4200/vendor.js:37612:63)
    at R3Injector.get (http://localhost:4200/vendor.js:37373:33)
    at injectInjectorOnly (http://localhost:4200/vendor.js:23517:33)